package.json

```diff
@@ -21,13 +21,12 @@
   },
   "homepage": "https://github.com/driftyco/ionic-module-template#readme",
   "devDependencies": {
```

Comment:
So I give up getting this to work with SCSS and external templates. Rangle.io seems to hint at using `@ngtools/webpack` for AoT compilation, in order to get such advanced features working out of the box.

I simply don't have the required time right now, to get that to work.